Clinical-Genomics/scout
# Submit SVs to MME as genes

fix #1583 

**ALERT: this is not the first time I've been having problems while testing stuff using the `requests` library, because of the existence of a `scout.utils.requests` module. To avoid problems and confusion in general, in this PR I'm renaming the scout requests to `scout.utils.scout_requests`.**

**Problem: at the moment it is not possible to submit SV variants to MatchMaker.**

1. If a patient has only one pinned variant (SV) and no phenotype data:
   The patient submission fails -> The patient is not compliant the the MME API error is returned

1. If a patient has SNVs and SVs (max 3 variants in total):
  Only SNVs are submitted

This PR makes it possible to submit genotype features for pinned SV variants when: 
**A:** SV variant contains max 3 genes.
**B:** Then: only gene names are submitted along with patient phenotype info, but the variant submission works.

**How to test**:
1. Go to hasta-stage server and update your user by granting yourself the authority to send patients to PatientMatcher: 
``` 
scout update user -r mme_submitter your.email@mail.com
``` 
1. Using the current master branch on scout-stage, test submitting one or more patients (don't forget to include hpo terms and/omim diagnoses) to confirm the conditions in 1. and 2.

1. Install this branch on clinical-db stage (the bash update-scout-stage script has some problems in the end so you should manually restart the server after installing the branch with the `supervisorctl restart scout-stage`.

1. Make sure you can submit also SV variants as described in **A** and **B**

**Expected outcome**:
The functionality should be working
**Please remove patients submitted to stage at the end of the test!**

**Review:**
